openpype/modules/deadline/plugins/publish/collect_pools.py

```diff
@@ -9,11 +19,12 @@
 
 class CollectDeadlinePools(pyblish.api.InstancePlugin,
                            OpenPypePyblishPluginMixin):
-    """Collect pools from instance if present, from Setting otherwise."""
+    """Collect pools from instance or attributes, from Setting otherwise."""
```

Comment:
What attributes? The collector doesn't seem to be do anything special with 'attributes' except for whatever is defined on the instance via the publisher UI via its attribute definitions.

There isn't really a **or** here because the behavior is singular.

Maybe better worded is removing the word "instance" and stating:

```suggestion
    """Collect pools from attributes or from Setting otherwise."""
```

However, to me collecting from the instance means collecting from whatever attributes are defined for it - so collecting attributes == collecting from the instance.